app/src/main/java/com/android/sample/ui/calendar/DayCalendarScreen.kt

```diff
@@ -132,20 +160,35 @@ fun HourBlock(hour: Int, workouts: List<ColoredWorkout>) {
  * @param coloredWorkout The colored workout to display, which includes workout name and color.
  */
 @Composable
-fun WorkoutItem(coloredWorkout: ColoredWorkout) {
-  Card(
-      modifier =
-          Modifier.fillMaxWidth()
-              .padding(vertical = 4.dp, horizontal = 30.dp)
-              .testTag("WorkoutCard")
-              .clickable { /*navigate to edit or start workout screen*/},
-      colors = CardDefaults.cardColors(containerColor = coloredWorkout.backgroundColor),
-      shape = MaterialTheme.shapes.medium) {
-        Row(
-            modifier = Modifier.fillMaxWidth().padding(16.dp),
-            horizontalArrangement = Arrangement.SpaceBetween) {
-              Text(text = coloredWorkout.workout.name, fontWeight = FontWeight.Bold)
-              Text(text = formatTime(coloredWorkout.workout.date))
-            }
-      }
+fun WorkoutItem(coloredWorkout: DayColoredWorkout, navigationActions: NavigationActions) {
+  val shape =
+      RoundedCornerShape(topStart = 15.dp, topEnd = 5.dp, bottomStart = 5.dp, bottomEnd = 15.dp)
+  Row() {
+    Spacer(modifier = Modifier.width(30.dp))
+    Card(
+        modifier =
+            Modifier.padding(horizontal = 50.dp)
+                .testTag("WorkoutCard")
+                .clickable {
+                  navigateToWorkoutScreen(
+                      coloredWorkout.workout, coloredWorkout.viewModel, navigationActions)
+                }
+                .shadow(4.dp, shape = shape),
+        colors = CardDefaults.cardColors(containerColor = coloredWorkout.backgroundColor),
+        shape = shape) {
+          Row(
+              modifier = Modifier.fillMaxWidth().padding(vertical = 2.dp, horizontal = 16.dp),
+              horizontalArrangement = Arrangement.SpaceBetween) {
+                Text(
+                    text = coloredWorkout.workout.name,
+                    fontFamily = OpenSans,
+                    fontSize = 18.sp,
```

Comment:
Same here and i think there might already be a font size saved that is about 17.sp or 16.sp if i remember correctly and I think it would be better if font sizes are similar between screens but again that's just my opinion